### PR TITLE
+ interactive side-menu' site without page reloads;

### PR DESCRIPTION
**Pull request by RomanSaiko**
### DOM - practise

[Demo](https://romansaiko.github.io/kottans-dom-practise/) |
[Code base](https://github.com/RomanSaiko/kottans-dom-practise)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/RomanSaiko/dom-practise/js/scripts.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        city: 'Amsterdam',
+        country: 'Netherlands',
+        region: 'North Holland',
+        population: '872,680',
+        area: '219.32',
+        website: 'https://www.amsterdam.nl/en/'
+    }
+];
+
+let renderPage = () => {
```
Comment (madmaxWMFU) [inline submissions/RomanSaiko/dom-practise/js/scripts.js:40]: Must be `const`

File: submissions/RomanSaiko/dom-practise/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        article.innerHTML = infoBlock;
+    }
+
+    createInfoBlock(0);
+
+    iconMenu.addEventListener('click', handleMenuToggle);
+    navMenu.addEventListener('click', changeInfo);
+}
+
+document.addEventListener('DOMContentLoaded', renderPage);
```
Comment (madmaxWMFU) [inline submissions/RomanSaiko/dom-practise/js/scripts.js:80]: In this function, I would leave the function calls and event handlers.
```
    createInfoBlock(0);
    iconMenu.addEventListener('click', handleMenuToggle);
    navMenu.addEventListener('click', changeInfo);
```

File: submissions/RomanSaiko/dom-practise/js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        iconMenu.classList.toggle('open');
+        if (iconMenu.classList.contains('open')) {
+            sidebarMenu.classList.add('open');
+        }
+        else {
+            sidebarMenu.classList.remove('open');
+        }
+    };
+
+    const changeInfo = (e) => {
```
Comment (madmaxWMFU) [inline submissions/RomanSaiko/dom-practise/js/scripts.js:56]: You can use [destructuring](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/%D0%94%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D1%96%D1%8F)

**Review by madmaxWMFU — CHANGES_REQUESTED** [review]
Good job!
See comments below

File: submissions/RomanSaiko/dom-practise/js/scripts.js (hunk 3 above)
Comment (RomanSaiko) [reply submissions/RomanSaiko/dom-practise/js/scripts.js:56]: Hi, are you talking about destructuring for lines 64-69?

**Comment by RomanSaiko** [comment]
@madmaxWMFU, check please

File: submissions/RomanSaiko/dom-practise/js/scripts.js (hunk 3 above)
Comment (madmaxWMFU) [reply submissions/RomanSaiko/dom-practise/js/scripts.js]: Something like this
```
const changeInfo = ({target}) => {
    const id = target.dataset.id;
```

**Comment by madmaxWMFU** [comment]
Make in one media query:
```
@media (max-width: 600px) {
  .aside-menu {
    transform: translateX(-200px);
  }

  .menu-handler {
    display: block;
  }

  .info-block {
    right: calc(50% - 150px);
  }
}
```

File: submissions/RomanSaiko/dom-practise/js/scripts.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+const content = [
```
[the remark this answers is not in the document]
Comment (MaxKalashnyk) [reply submissions/RomanSaiko/dom-practise/js/scripts.js:1]:
```suggestion
const citiesData = [
```

**Review by MaxKalashnyk — COMMENTED** [review]
LGTM! Look at my minor suggestions, please

**Review by MaxKalashnyk — APPROVED** [review]

**Pull request merged**